expression/expr_to_pb_test.go

```diff
@@ -147,7 +147,6 @@ func (s *testEvaluatorSuite) TestColumn2Pb(c *C) {
 	colExprs = append(colExprs, dg.genColumn(mysql.TypeEnum, 3))
 	colExprs = append(colExprs, dg.genColumn(mysql.TypeGeometry, 4))
 	colExprs = append(colExprs, dg.genColumn(mysql.TypeUnspecified, 5))
-	colExprs = append(colExprs, dg.genColumn(mysql.TypeDecimal, 6))
```

Comment:
Why remove this?

Reply:
Since if `mysql.TypeDecimal` is replaced into `mysql.TypeUnspecified`, the test will duplicate.

Reply:
Maybe we can change this TypeDecimal to TypeNewDecimal?

Reply:
Emmmm... The test for `mysql.TypeNewDecimal` has been added in the following code. Add it will also duplicate.